<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallMapItemWidget</class>
 <widget class="QWidget" name="InstallMapItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InstallMapItemWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" rowspan="2">
      <widget class="QLabel" name="mapDescriptionLabel">
       <property name="text">
        <string>Map Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="installPushButton">
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="usePushButton">
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="mapTitleLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
